src/engine/strat_engine/filesystem.rs

```diff
@@ -64,6 +82,32 @@ impl StratFilesystem {
         self.thin_dev.id()
     }
 
+    /// Return an extend size for the filesystem over the thin device.
+    /// TODO: returning the current size will double the space provisoned to
+    /// the filesystem.  We should determine if this is a reasonable value.
+    fn extend_size(&self, current_size: DataBlocks) -> DataBlocks {
+        DataBlocks(*current_size * 2)
+    }
+
+    /// Get the mount_point for this filesystem
+    /// TODO Replace this code with something less brittle
+    pub fn get_mount_point(&self) -> EngineResult<PathBuf> {
+        let output = try!(Command::new("df")
+                              .arg("--output=target")
```

Comment:
the `df` man page doesn't show `--output=target` as a valid option, and it's not working for me on the command-line either...?